### PR TITLE
Update metadata and audio quality rules for Loved

### PR DESCRIPTION
**Pull request by zeusminus**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

`SKIP_OUTDATED_CHECK` (broken atm)

### CODE REVIEW

File: wiki/Community/Project_Loved/en.md [hunk 1]
```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
 
 Beatmaps entering the Loved category are required to follow some additional rule sets:
 
-- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Partially the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
```
Comment (peppy) [inline wiki/Community/Project_Loved/en.md:35]: I don't know how to read this section.

Comment (zeusminus) [reply wiki/Community/Project_Loved/en.md:35]: The following sub-bulletpoints were meant to underline which sections of the mentioned portion of RC the beatmap being moved to Loved should abide by. If it's not well-enough formed, I can try phrasing it differently.

File: wiki/Community/Project_Loved/en.md [hunk 2]
```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
...
+  - Song title, artist and source should be correct in all cases.
+  - Beatmap tags should properly list all contributors (guest difficulty creators, storyboarders, etc.).
 - [Song Content Rules](/wiki/Rules/Song_Content_Rules)
 - [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations)
 
-Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap's audio files should be from the highest quality source available while having an average bit rate no greater than 192kbps.
+Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap's audio files should be at least 128kbps mp3 equivalent.
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:41]: slightly assuming on what "mp3 equivalent" is supposed to mean

```suggestion
Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap audio files should have an average bitrate of at least 128kbps.
```

File: wiki/Community/Project_Loved/en.md (hunk 1 above)
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:35]:
```suggestion
- [Some points from the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
```

File: wiki/Community/Project_Loved/en.md [hunk 3]
```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
 
 Beatmaps entering the Loved category are required to follow some additional rule sets:
 
-- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Partially the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+  - Song title, artist and source should be correct in all cases.
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:36]:
```suggestion
  - Song title, artist, and source should be correct in all cases.
```

File: wiki/Community/Project_Loved/en.md [hunk 4]
```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
 
 Beatmaps entering the Loved category are required to follow some additional rule sets:
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:33]:
```suggestion
Beatmaps entering the Loved category are required to follow some additional rulesets:
```

File: wiki/Community/Project_Loved/en.md (hunk 2 above)
Comment (mousesalt) [reply wiki/Community/Project_Loved/en.md:41]: "128 kbps" wording here sound like the audio should be in loved section, when in order ranking criteria, 192 kbps is the higher audio can be used.

Comment (Hiviexd) [reply wiki/Community/Project_Loved/en.md:41]: we always allow bloated audios to be loved as-is, and allow ones that are above 192kbps to be loved under certain conditions, so including the audio RC here would be misleading. 
our only requirement is for the audio to be at least 128kbps.

**Review by peppy — APPROVED** [review]

File: wiki/Community/Project_Loved/en.md [hunk 5]
```diff
@@ -30,13 +30,15 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 
 Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-progress-and-pending) maps meet these criteria, few are voted into the Loved category due to the more selective nomination and voting requirements.
 
-Beatmaps entering the Loved category are required to follow some additional rule sets:
+Beatmaps entering the Loved category are required to follow some additional rulesets:
 
-- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Some points from the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:35]:
```suggestion
- [Some points from the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata):
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**